fcrepo6/rootfs/etc/cont-init.d/03-fcrepo-setup.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/with-contenv bash
+set -e
+
+function execute_sql_file {
+    execute-sql-file.sh \
+        --driver "${FCREPO_PERSISTENCE_TYPE}" \
```

Comment:
See notes above for fcrepo4, May or may not be relevant to this since `repository.json` is no longer in use.